docs/tutorials/getting-started/tut-setup-dev.md

```diff
@@ -324,8 +324,9 @@ Then, make sure that `demisto-sdk` has been installed automatically by the boots
 Use demisto-sdk -h to see the available commands.
 ```
 
-Now, run the `demisto-sdk lint` command on the folder `Integrations/HelloWorld` using the `-d` option. It will run both the [linters](../../howtos/integrations/linting) and [pytest](../../howtos/integrations/unit-testing):
-
+Now, run the `demisto-sdk lint` command on the folder `Packs/HelloWorld/Integrations/HelloWorld` using the `-d` option, or if you have
+multiple changes you can use the `demisto-sdk lint -g` option which will detect the changes you've made and will run our linter based on the changed files in your cloned repo.
```

Comment:
I would change it to be something like `demisto-sdk lint -g` will run on all the committed files on your branch

Reply:
multiple changes can be misleading here